media-gfx/img2pdf/img2pdf-0.4.0.ebuild

```diff
@@ -0,0 +1,38 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{7,8} )
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1
+
+DESCRIPTION="Losslessly convert raster images to PDF"
+HOMEPAGE="https://gitlab.mister-muffin.de/josch/img2pdf"
+SRC_URI="mirror://pypi/${PN:0:1}/${PN}/${P}.tar.gz"
+
+LICENSE="LGPL-3+"
+SLOT="0"
+KEYWORDS="~amd64"
+
+BDEPEND="test? (
+	dev-python/numpy[${PYTHON_USEDEP}]
+	dev-python/pdfrw[${PYTHON_USEDEP}]
+	dev-python/pytest[${PYTHON_USEDEP}]
+	dev-python/scipy[${PYTHON_USEDEP}]
+	<media-gfx/imagemagick-7.0.0[jpeg,jpeg2k,png,q8,q32,tiff]
```

Comment:
This is really unfortunate. Avoidable at all?

Reply:
I tried to use imagemagick-7 but first problem is that formatDescription change, for example, from "Portable Network Graphics" to "PNG" so it would be necessary to patch.
Even doing this other tests fail.